cypress/integration/internal/change-user-permissions.spec.js

```diff
@@ -0,0 +1,41 @@
+const { setUp, tearDown } = require('../../support/setup');
+
+describe('change internal user permissions as B&D user', () => {
+  before(() => {
+    tearDown();
+    setUp('barebones');
+  });
+
+  after(() => {
+    tearDown();
+  });
+
+  it('searches for user by email address', () => {
+    cy.visit(Cypress.env('ADMIN_URI'));
+    cy.fixture('users.json').then(users => {
+      cy.get('input#email').type(users.billingAndData);
+      const EMAIL_ADDRESS = users.environmentOfficer;
+      cy.get('#password').type(Cypress.env('DEFAULT_PASSWORD'));
+      cy.get('.govuk-button.govuk-button--start').click();
+      cy.get('#query').type(EMAIL_ADDRESS);
+      cy.get('.search__button').click();
+      cy.get('.govuk-grid-column-full > .govuk-heading-m').should('have.text', 'Users');
+      cy.get('.govuk-list > li').should('contain.text', EMAIL_ADDRESS);
+      cy.get('.govuk-list .govuk-link').click();
+
+      describe('navigates to the user page', () => {
+        cy.url().should('include', '/status');
```

Comment:
`/status` may also appear in the URL for other pages. Can you make this more specific please?

E.g. you could add
```
cy.url().should('include', '/user/');
```